documentation/book/assembly-getting-started-strimzi-admin.adoc

```diff
@@ -0,0 +1,13 @@
+// This assembly is included in the following assemblies:
+//
+// getting-started.adoc
+
+[id='assembly-getting-started-strimzi-admin-{context}']
+= Strimzi Administrator
+
+{ProductName} adds several custom resources.
+By default, only {ProductPlatformName} cluster administrators users will have the rights to create, edit and delete these resources.
```

Comment:
"these" is not clear here to me. Instead, the next one "{ProductName} resources" is clear.

Reply:
It refers to the sentence above. Do you think its not clear?

Reply:
I miss the first sentence :dagger: